src/Lib/LaramieHelpers.php

```diff
@@ -69,19 +69,21 @@ public static function formatListValue($field, $value, $isShowUnsupporteMessage
             case 'currency':
             case 'month':
             case 'tel':
-            case 'radio':
             case 'url':
             case 'week':
             case 'time':
             case 'checkbox':
             case 'computed':
                 return $value;
+            case 'radio':
             case 'select':
-                if (is_array($value)) {
-                    return implode(', ', $value);
-                }
+                $options = collect(data_get($field, 'options', []));
+                $value = is_array($value) ?: [ $value ];
```

Comment:
If `$value` _is_ an array, I think it would get set to `true`. I think we'd need `$value = is_array($value) ? $value : [$value];`